galasa-ui/messages/en.json

```diff
@@ -282,5 +284,75 @@
     "from": "From",
     "to": "To",
     "duration": "Duration"
+  },
+  "SearchCriteriaContent": {
+    "description": "Edit search criteria to describe the test results you wish to view",
+    "table": {
+      "columnName": "Column Name",
+      "allowedValues": "Allowed Values"
+    },
+    "fields": {
+      "runName": {
+        "label": "Test Run Name",
+        "placeholder": "any",
+        "description": "Type the name of one test run. An exact match is searched for."
+      },
+      "requestor": {
+        "label": "Requestor",
+        "placeholder": "any",
+        "description": "Type the name of a requestor. An exact match is searched for."
```

Comment:
We no longer insist on the requestor being an exact match - it is case-insensitive now, so it would be great to update this message (same applies for the German translation)